kitsune/sumo/jinja2/base.html

```diff
@@ -182,12 +182,6 @@ <h2 class="logo">
       <ul>
         {{ aux_nav(user, hide_aaq_link) }}
 
-        {% if not hide_locale_switcher %}
```

Comment:
How is this relevant to the OIDC backend ?

Reply:
It's not. I will remove this commit from this PR